✨Source Open Exchange Rates: Migrate Python CDK to Low-code CDK

btkcodedev opened the pull request:
## What
*Migrating Source Open Exchange Rates to Low-Code CDK*
Closes #30906

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results
![image](https://github.com/airbytehq/airbyte/assets/47392334/ebffaa80-2ab7-40ed-b675-2db4e781a24d)


</details>


## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Open Exchange Rates</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

btkcodedev commented:
Test results are a little stumbled upon incremental reads, but full refresh is passing
![image](https://github.com/airbytehq/airbyte/assets/47392334/6a9589e9-db7b-425a-8fd9-8a7323d6ebae)

btkcodedev commented:
config
{
  "start_date": "2023-08-01",
  "base": "USD",
  "app_id": "0f492965c4e24af7bc1f5a6ddc63acbb"
}

Open-temp-creds

btkcodedev commented:
Ready for review comments please :)

marcosmarxm commented:
Thanks for the contribution @btkcodedev. I added it to the team backlog and we're planning to start next week the review process.

marcosmarxm commented on airbyte-integrations/connectors/source-open-exchange-rates/acceptance-test-config.yml:
```diff
@@ -1,9 +1,12 @@
   spec:
     tests:
       - spec_path: "source_open_exchange_rates/spec.yaml"
+        backward_compatibility_tests_config:
+          disable_for_version: 0.1.0
```
If the previous version isn't backward compatible you need to release a major version, change to 1.0.0 as a breaking change. Why are you update the schema and specification of this connector?

marcosmarxm reviewed (changes requested):
Left a review to understand the breaking change.

btkcodedev commented:
Oops, that was unwanted, Fixed the schema and reverted that skipping of test :)
New test result: All pass except inc sync

![image](https://github.com/airbytehq/airbyte/assets/47392334/dcf0cd3e-965f-4fae-8e33-6b65afaa73a9)
